### PR TITLE
Bump CNAO to v0.36.0

### PR DESCRIPTION
**Pull request by tiraboschi**
Signed-off-by: Simone Tiraboschi <stirabos@redhat.com>

**Release note**:
```release-note
Bump CNAO to v0.36.0
```

### CODE REVIEW

File: deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1299,6 +1299,8 @@ spec:
                     Manufacturer: KubeVirt
                     Product: None
                 - name: MACHINETYPE
+                - name: HCO_KV_IO_VERSION
```
Comment (phoracek) [inline deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml:1302]: Is this related?

Comment (tiraboschi) [reply deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml:1302]: No, we added it in #528 and we forgot to regenerate the CSV there
